packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -36,15 +43,21 @@ export function createServer() {
 
   const addMeteredEndpoint = (
     endpoint: Endpoints,
-    handler: (req: Request, res: Response) => Promise<void>
+    handler: (req: Request, res: Response) => Promise<void>,
+    method: 'post' | 'get' = 'post'
```

Comment:
I don't think there are any `GET` endpoints defined because all the new requests use the body to send their parameters. Why is handling of `GET` added here now?

Reply:
I think just for forwards compatibility. An earlier version of this PR used GET for the status endpoint and it doesn't seem that there's any harm in keeping this.